packages/turf-transform-rotate/README.md

```diff
@@ -0,0 +1,51 @@
+# @turf/transform-rotate
+
+# rotate
+
+Rotates any geojson Feature or Geometry of a specified angle, around its `centroid` or a given `pivot` point
+
+**Parameters**
+
+-   `geojson` **[Geometry](http://geojson.org/geojson-spec.html#geometry)&lt;any> \|
+  [Feature](http://geojson.org/geojson-spec.html#feature-objects)&lt;any>** GeoJSON object to be rotated
+-   `angle` **[number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)** of 
+rotation, from North in decimal degrees, positive clockwise 
+-   `pivot` **([Geometry](http://geojson.org/geojson-spec.html#geometry) \| [Feature](http://geojson.org/geojson-spec.html#feature-objects)&lt;[Point](http://geojson.org/geojson-spec.html#point)> | [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)&lt;[number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)>)** point around which the rotation will be performed 
+(optional, default `centroid`)
+
+**Examples**
+
+```javascript
+var poly = turf.polygon([[0,29],[3.5,29],[2.5,32],[0,29]]);
+var rotatedPoly = turf.rotate(poly, 85);
+
+//addToMap
+var addToMap = [rotatedPoly];
```

Comment:
Might want to include "before" & "after", simply seeing 1 polygon won't show you much (might want to change the colour of the fill).